docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc

```diff
@@ -33,38 +35,42 @@ You can also set the following properties to change the behavior of the pre-auth
 | false
 
 | failOnError
-| If true, disallow login if the header is not set or the user does not exist. If false, fall through to other mechanisms (basic auth, form login, and so on.)
+| If `true`, disallow login if the header is not set or the user does not exist.
+If `false`, fall through to other mechanisms (basic auth, form login, and so on).
 | false
 
 | userHeader
-| The HTTP header that will specify the user to authenticate as.
+| The HTTP header that specifies the user to authenticate as.
 | X-Remote-User
 
 | credentialsHeader
-| A comma-separated list of credentials to associate with the principal. If specified, the header must be set for authentication to be successful.
-| n/a
+| A comma-separated list of credentials to associate with the principal.
+If specified, the header must be set for authentication to be successful.
+|
```

Comment:
I'm going to suggest we use "blank," in italics, for parameters that don't have default values. If you agree, I can go through and add suggestions to the appropriate lines in this PR. :)

```suggestion
| _blank_
```

Reply:
I ended up removing the n/a, as in the context of the item being a list, it didn't seem necessary to include anything in the field. 
I like your suggestion, though, which would work elsewhere in the docs. Let's make it more specific so that it works for localization. _no default value_

Reply:
I've added it to the other cell in this table. When you review my updates, please do go through and make other suggestions.